### ISSUE TITLE
have you tried with NVCaffe-0.16

### ISSUE THREAD

**Issue by CFAndy** [shicai/MobileNet-Caffe#10]
I tried but only reach 0.63 top-1. Have you shuffled your data for each epoch?
609 01:12:10.964810 19888 caffe.cpp:300] Batch 998, accuracy = 0.66
I0609 01:12:10.964838 19888 caffe.cpp:300] Batch 998, accuracy_top5 = 0.84
I0609 01:12:10.964844 19888 caffe.cpp:300] Batch 998, loss = 1.49327
I0609 01:12:11.079627 19888 caffe.cpp:300] Batch 999, accuracy = 0.56
I0609 01:12:11.079668 19888 caffe.cpp:300] Batch 999, accuracy_top5 = 0.82
I0609 01:12:11.079674 19888 caffe.cpp:300] Batch 999, loss = 1.9213
I0609 01:12:11.079679 19888 caffe.cpp:305] Loss: 1.53589
I0609 01:12:11.079704 19888 caffe.cpp:317] accuracy = 0.633339
I0609 01:12:11.079715 19888 caffe.cpp:317] accuracy_top5 = 0.852882
I0609 01:12:11.079725 19888 caffe.cpp:317] loss = 1.53589 (* 1 = 1.53589 loss)
-Andy

**Comment by shicai** [comment]
yes, i did use data shuffling for each epoch.
btw, is this the result from the model you trained from scratch?

**Comment by CFAndy** [comment]
Yes, it is.

**Comment by AlexofNTHU** [comment]
@CFAndy Can you share your training settings including solver and training prototxt? How much ms with what model of GPU did you spend for single image inference?

**Comment by leon-liangwu** [comment]
@CFAndy @shicai  I tried to train mobilenet from scratch using nvcaffe. But I got that the training loss in declining while the testing loss is climbing. Could you tell me what may be the course?
